scripts/Tooltip.js

```diff
@@ -116,7 +118,12 @@ class Tooltip {
   // * font awesome icon
   // * url
   _getIconData(icon) {
-    return { icon, iconType: icon ? this._reg.faIcon.test(icon.trim()) : true, iconSize: this._fontSize };
+    return {
+      icon: (icon&&this._reg.html.test(icon.trim()))?icon.trim().substring(1) : icon,
```

Comment:
Codacy has a fix for the issue: [Operator '&&' must be spaced.](https://app.codacy.com/gh/bmarian/token-tooltip-alt/pullRequest?prid=7484852)
```suggestion
      icon: (icon && this._reg.html.test(icon.trim()))?icon.trim().substring(1) : icon,
```